src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -727,6 +735,15 @@ class LicenceService(
         )
       )
     )
+
```

Comment:
Is it possible that the createdBy AND COM Email are null? If so, it might be an idea to check if both are messing before calling the method, and logging the lack of email addresses using the logger before calling return on the current method.